### PR TITLE
[MOSIP-29662] changed protocol logic in datashare

### PR DESCRIPTION
**Pull request by GOKULRAJ136**

### CODE REVIEW

**Review by balaji-alluru — APPROVED** [review]

File: data-share/data-share-service/pom.xml [hunk 1]
```diff
@@ -5,11 +5,11 @@
   <parent>
     <groupId>io.mosip.datashare</groupId>
     <artifactId>data-share</artifactId>
-    <version>1.1.5.3-P2</version>
+    <version>1.1.5.3-P3</version>
```
Comment (sowmya695) [inline data-share/data-share-service/pom.xml:8]: update to 1.1.5.3-P3-SNAPSHOT

File: data-share/data-share-service/pom.xml [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
   </parent>
 
   <artifactId>data-share-service</artifactId>
-  <version>1.1.5.3-P2</version>
+  <version>1.1.5.3-P3</version>
```
Comment (sowmya695) [inline data-share/data-share-service/pom.xml:12]: update to 1.1.5.3-P3-SNAPSHOT

File: data-share/pom.xml [hunk 3]
```diff
@@ -5,11 +5,11 @@
   <parent>
     <groupId>io.mosip.datashare</groupId>
     <artifactId>durian</artifactId>
-    <version>1.1.5.3-P2</version>
+    <version>1.1.5.3-P3</version>
```
Comment (sowmya695) [inline data-share/pom.xml:8]: update to 1.1.5.3-P3-SNAPSHOT

File: data-share/pom.xml [hunk 4]
```diff
@@ -5,11 +5,11 @@
...
   </parent>
   
   <artifactId>data-share</artifactId>
-  <version>1.1.5.3-P2</version>
+  <version>1.1.5.3-P3</version>
```
Comment (sowmya695) [inline data-share/pom.xml:12]: update to 1.1.5.3-P3-SNAPSHOT

File: pom.xml [hunk 5]
```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>	
 	<groupId>io.mosip.datashare</groupId>
 	<artifactId>durian</artifactId>
-	<version>1.1.5.3-P2</version>
+	<version>1.1.5.3-P3</version>
```
Comment (sowmya695) [inline pom.xml:7]: update to 1.1.5.3-P3-SNAPSHOT

**Review by sowmya695 — CHANGES_REQUESTED** [review]
Please  do the requested changes

File: data-share/data-share-service/pom.xml (hunk 1 above)
Comment (GOKULRAJ136) [reply data-share/data-share-service/pom.xml]: updated to 1.1.5.3-P3-SNAPSHOT version

File: data-share/data-share-service/pom.xml (hunk 2 above)
Comment (GOKULRAJ136) [reply data-share/data-share-service/pom.xml]: updated to 1.1.5.3-P3-SNAPSHOT version

File: data-share/pom.xml (hunk 3 above)
Comment (GOKULRAJ136) [reply data-share/pom.xml]: updated to 1.1.5.3-P3-SNAPSHOT version

File: data-share/pom.xml (hunk 4 above)
Comment (GOKULRAJ136) [reply data-share/pom.xml]: updated to 1.1.5.3-P3-SNAPSHOT version

File: pom.xml (hunk 5 above)
Comment (GOKULRAJ136) [reply pom.xml]: updated to 1.1.5.3-P3-SNAPSHOT version

**Comment by GOKULRAJ136** [comment]
> Please do the requested changes

updated to 1.1.5.3-P3-SNAPSHOT version

**Review by sowmya695 — APPROVED** [review]

**Pull request merged**